src/libs/reportUtils.js

```diff
@@ -102,6 +102,26 @@ function isDefaultRoom(report) {
     ], lodashGet(report, ['chatType'], ''));
 }
 
+/**
+ * Get either the policyName or domainName the chat is tied to
+ * @param {Object} report
+ * @param {Object} policiesMap must have onyxkey prefix (i.e 'policy_') for keys
+ * @returns {String}
+ */
+function getDefaultRoomSubtitle(report, policiesMap) {
+    if (!isDefaultRoom(report)) {
+        return '';
+    }
+    if (report.chatType === CONST.REPORT.CHAT_TYPE.DOMAIN_ALL) {
+        return report.reportName.substring(1);
```

Comment:
Why do we need the substring(1) call? Isn't `report.reportName` sufficient?

Reply:
The reportName has the `#` in it so this just gets rid of that.

Reply:
Solid, sounds good!

Reply:
That sounds good, but I think a comment refering that would be nice, just like us whom ever is reading this code might not know and get a bit confused on to the why. Also if at any point we don't have the `#` before it might make it easier to search for it.

Reply:
That's a good point, I'll leave one in!